docs/features.md

```diff
@@ -327,11 +351,13 @@ You can use both sync and async functions for middlewares!
 ```python
 @app.before_request("/")
 async def hello_before_request(request):
+    request.headers["before"] = "sync_before_request"
     print(request)
 
 
 @app.after_request("/")
 def hello_after_request(request):
+    request.headers["after"] = "sync_after_request"
     print(request)
```

Comment:
```suggestion
def hello_after_request(response: Response):
    response.headers["after"] = "sync_after_request"
    print(response)
```

Reply:
Please don't forget to change the param to a `Response` in the after middleware. As we already executed the route method, we got rid of the `Request` and now have a `Response`, which is what we can modify in the after middleware.

Reply:
done!